src/js/components/Button/doc.js

```diff
@@ -251,6 +270,30 @@ export const themeDoc = {
     type: '{}',
     defaultValue: undefined,
   },
+  'button.badge.background': {
+    description: `The background of the badge.`,
+    type: `string | { dark: string, light: string } | {
+      color: { dark: string, light: string } | string,
+      dark: bool,
+      image: string,
+      position: string,
+      opacity: bool | string,
+      repeat: no-repeat | repeat,
+      size: cover | contain | string
+    }`,
+    defaultValue: 'brand',
+  },
+  'button.badge.max': {
+    description: `The max number to display in the badge. If max is 9 and badge 
+    has value of 10, 9+ will be displayed.`,
+    type: 'number',
+    defaultValue: undefined,
+  },
+  'button.badge.size.medium': {
+    description: `The minimum width and height of the badge.`,
```

Comment:
Minimum when containing content? What if badge={true}?

Reply:
Good clarification. Adding that.